run.py

```
@@ -0,0 +1,4 @@
+import uvicorn
+
+if __name__ == "__main__":
+    uvicorn.run("app.main:app", host="0.0.0.0", port=8000, reload=True, reload_dirs=["app", "templates"])
```

Ruff (0.11.9)

No newline at end of file

Add trailing newline

(W292)